EdurecaInClassProject/src/app/components/content/content.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-content',
+  templateUrl: './content.component.html',
+  styleUrls: ['./content.component.css']
+})
+export class ContentComponent implements OnInit {
+
+  constructor() { }
```

Comment:
This is the empty component, where it is getting used exactly?

Reply:
in this component, I have added a header footer components